plugins/database/mysql/mysql_test.go

```diff
@@ -44,6 +45,40 @@ func TestMySQL_Initialize(t *testing.T) {
 	}
 }
 
+// TestMySQL_Initialize_CloudGCP validates the proper initialization of a MySQL backend pointing
+// to a GCP CloudSQL MySQL instance. This expects some external setup (exact TBD)
+func TestMySQL_Initialize_CloudGCP(t *testing.T) {
+	// hardcode data for now
+	url := os.Getenv("CONN_URL")
+	creds := os.Getenv("CREDS")
+
+	tests := map[string]struct {
+		req dbplugin.InitializeRequest
+	}{
+		"normal": {
+			req: dbplugin.InitializeRequest{
+				Config: map[string]interface{}{
+					"connection_url": url,
+					"auth_type":      "iam", // authTypeIAM,
```

Comment:
should we use the constant here?

Reply:
I thought about this, and had some philosophical thoughts about how "this test _should_ break if someone changes the constant value", but I think that argument is verrrry marginal.

Reply:
I think that now that we only specify the constant once, it's okay to just reference the constant in the tests.